packages/medulas-react-components/src/components/Image/index.stories.tsx

```diff
@@ -1,14 +1,35 @@
 import { storiesOf } from '@storybook/react';
 import React from 'react';
 import Image from './index';
+import CircleImage from './CircleImage';
 import { Storybook } from '../../utils/storybook';
 import iovLogo from './assets/iov-logo.png';
+import download from '../../theme/assets/download.svg';
+import Grid from '../Grid';
+import GridItem from '../GridItem';
+import theme from '../../theme/utils/mui';
```

Comment:
Can we use `useTheme` here as well?

Reply:
Not possible because of different versions of react (in storybook itself: 16.8.6 and system wide: 16.9.0-alpha.0) and this will it possible to use hooks (well known "Invalid hook call. Hooks can only be called inside of the body of a function component." issue)